frontend/src/components/pod/List.tsx

```diff
@@ -39,7 +47,7 @@ export default function PodList() {
       <SimpleTable
         rowsPerPage={[15, 25, 50]}
         filterFunction={filterFunc}
-        errorMessage={Pod.getErrorMessage(error)}
+        errorMessage={Pod.getErrorMessage(error as ApiError)}
```

Comment:
The thing is we want to allow the error to be an optional entity to the Component thus error has types undefined | null | ApiError.
The function Pod.getErrorMessage takes a type ApiError | null as argument that is why we need a explicit typecast here and we are safe here because there is a value check ultimately inside the getErrorMessage method

Reply:
If Pod.getErrorMessage should handle undefined, then the type should be changed there to say it can. Another way to fix it is to consider if error should be `APIError | null | undefined`... and instead just be `APIError | null` (maybe looking at the hook above to be non-conditional could clarify the types here).

Reply:
Fixed with how we were handling apiPods